src/charts/bar.js

```diff
@@ -317,6 +328,81 @@ define(function(require) {
         }
 
         /**
+         * Draws and animates the bars along the x axis
+         * @param  {D3Selection} bars Selection of bars
+         * @return {void}
+         */
+        function drawAnimatedHorizontalBars(bars) {
+            // Enter + Update
```

Comment:
what about trying to combine some of the logic here? I see this divergent pattern as a potential problem in the future, where horizontal and vertical end up with different behaviors/features